### ISSUE TITLE
[CORE] Can't generate a new candidate

### ISSUE THREAD

**Issue by PapillonMcGill** [aces/Loris#6059]
**Describe the bug**
get `Uncaught DomainException: Values not configured properly for PSCIDGenerator. Please correct your configuration file.Length: `4`\nAlphabet: `0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ`\nMin: ``\nMax: `9999`\nLength: `CHA`\nGeneration: `sequential`\n in `

error reported on PSCID generation.

**To Reproduce**
Steps to reproduce the behavior (attach screenshots if applicable):
1. Go to 'Candidate -> new profile'
2. Set the proper values for the new candidate
3. Click on `Create`
4. button `gray-out` and nothing append, error in the log

**What did you expect to happen?**
A new candidate to be created and page update with candidate's profile

**Server Environment (if known):**
- LORIS Version: [aces/master, last commit at time]
- Linux distribution and Version: [e.g. Ubuntu 16.04]
- MySQL/MariaDB Version: [e.g. MySQL 5.7]

**Additional context**
ran `checkout aces/master`, `make` and `php raisinbread_refresh.php`

config.xml file with proper value as indicated on docs/config/config.xml
```
   <!-- study variables -->
    <study>
        <PSCID>
            <!-- PSCID (Study Center ID) generation method possible options: sequential/random/user -->
            <generation>sequential</generation>
            <structure>
                <seq type="projectAbbrev"/>
                <seq type="alphanumeric" length="4" min='1' max='9999'/>  <!-- Ex: AAA1-->
            </structure>
        </PSCID>

        <ExternalID>
            <generation>random</generation>
            <structure>
                <seq type="static">BBB</seq>
                <seq type="numeric" length="4" min='1000' max='9999'/>
            </structure>
        </ExternalID>
```

**Comment by johnsaigle** [comment]
> Length: `CHA`

I think this is the bad value

**Comment by PapillonMcGill** [comment]
@johnsaigle This `CHA` is not from the config.xml file. I come from somewhere else.

**Comment by PapillonMcGill** [comment]
@ridz1208 @johnsaigle `CHA` in the bug description is coming from to project. I tried with another project (DCP from raisinbread) and `DCP` was in the length fields.

**Comment by johnsaigle** [comment]
@PapillonMcGill This problem seems to be fixed in another PR I have open, #6036. Could you test it and verify?
